auth/test/index.spec.ts

```diff
@@ -0,0 +1,114 @@
+import request from 'superagent';
```

Comment:
Can you create some more structure with folders and files? Split up shared code. Extract interfaces. Create a file for each spec/route.

Sadly, here are just a very few tests. Please make a good structure in this PR and tests can be added afterwards.

Reply:
Yes, I don't want to hold every test in a single file. This is a file for a first setup.

Reply:
But please do the structure right now. These tests must not contain not much, but structurewise it would be nice to see how you would organise the tests.